DevOps.Util/DevOpsHttpClient.cs

```diff
@@ -49,25 +50,75 @@ internal DevOpsHttpClient(AuthorizationToken authorizationToken = default, HttpC
             HttpClient = httpClient ?? new HttpClient();
         }
 
-        internal async Task DownloadFileAsync(string uri, Stream destinationStream)
+        private async Task DownloadWithProgress(HttpResponseMessage response, Stream destinationStream)
+        {
+            Console.Write($"Downloading...");
```

Comment:
I would generally keep `Console` out of this layer and put it in the surrounding code. Maybe consider a pattern where an interface is passed down and the impl can decide to log to `Console`